WordPress/src/main/res/layout/tooltip_above.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="vertical"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:gravity="center_horizontal">
+
+    <TextView
+        android:id="@+id/tooltip_message"
```

Comment:
Could you maybe extract all the common properties into a style? These 4 layouts are very similar.

Reply:
Good point, done here 48a3e9960bde6726ac891445a9f5d58d24cdf0eb